addons/digest/data/digest_data.xml

```diff
@@ -41,5 +41,18 @@
 </div>
             </field>
         </record>
+
+        <record id="digest_tip_mail_2" model="digest.tip">
+            <field name="group_id" ref='base.group_system'/>
+            <field name="sequence">0</field>
+            <field name="tip_description" type="html">
+<div>
+    <div style="font-size: 14px;">
+        To reach your full potential with Odoo, have a look at our <a href="https://www.odoo.com/page/tour">apps videos</a> and discover new tools to work better and faster.
+    </div><br/>
```

Comment:
Why 2 nested div and an unnecessary br ?

Reply:
Alright, modified it. :+1: